src/specs/api.v2.yaml

```diff
@@ -939,6 +938,77 @@ paths:
             application/json:
               schema:
                 $ref: ./models/HTTPError.v1.yaml
+  '/experiments/{experimentId}/gem2s':
+    post:
+      summary: Run gem2s
+      operationId: runGem2s
+      x-eov-operation-id: gem2s#run
+      x-eov-operation-handler: routes/gem2s
+      description: Triggers a new gem2s run
+      requestBody: 
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                paramsHash:
+                  type: string
+              required:
+                - paramsHash
+              additionalProperties: false
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPSuccess'
+        '400':
+          description: Bad Request
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+        '404':
+          description: Experiment not found
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPError'
+  /gem2sResults:
+    post:
+      operationId: receiveGem2sResponse
+      x-eov-operation-id: gem2s#response
+      x-eov-operation-handler: routes/gem2s
+      requestBody:
+        description: 'The results from the execution of a pipeline step, sent via SNS.'
+        required: true
+        content:
+          text/plain:
+            schema:
+              type: string
+          application/json:
+            schema:
+              type: object
+              properties: {}
+      responses:
```

Comment:
missing a 500 (I saw it in API v1, not sure if needed)

Reply:
it can, added the 500 again